CODEOWNERS

```diff
@@ -0,0 +1,73 @@
+# See also:
+# - https://github.com/blog/2392-introducing-code-owners
+# - https://help.github.com/articles/about-codeowners/
+
+# Each line is a file pattern followed by one or more owners.
+# Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
+
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
+
+etc/c/* @CyberShadow
+std/* @andralex
```

Comment:
> I should be on hook for all of std/, at least for the time being. Thanks!

@andralex AFAICT the CODEOWNERS file will ask for a reviewer for the last matching line, hence this line is a fall-through (and of course documentation).